### PR TITLE
style: improved styling to match Swarm branding

### PR DESCRIPTION
**Pull request by wendydv1989**

### CODE REVIEW

File: src/components/NavBar.tsx [hunk 1]
```diff
@@ -40,21 +39,21 @@ export default function SideBar(props: any) {
 
   return (
     <div>
-      <AppBar position="fixed" className={classes.appBar}>
-        <Toolbar style={{display: 'flex'}}>
+      <div style={{ display: 'fixed' }} className={classes.appBar}>
```
Comment (vojtechsimetka) [inline src/components/NavBar.tsx:42]: why is this changed to div? Also don't use `style` here please, instead add any styling into `appBar` object.

**Review by vojtechsimetka — APPROVED** [review]

**Comment by wendydv1989** [comment]
with love

File: src/components/SideBar.tsx [hunk 2]
```diff
@@ -2,46 +2,47 @@ import React from 'react';
 import { Link } from 'react-router-dom';
 
 import { createStyles, Theme, makeStyles } from '@material-ui/core/styles';
-import { ListItemText, ListItemIcon, ListItem, Divider, List, Drawer, Link as MUILink} from '@material-ui/core';
+import { ListItemText, ListItemIcon, ListItem, Divider, List, Drawer, Link as MUILink, ListItemSecondaryAction } from '@material-ui/core';
 import { OpenInNewSharp } from '@material-ui/icons';
-import { Activity, FileText, DollarSign, Share2, Settings } from 'react-feather';
+import { Activity, FileText, DollarSign, Share2, Settings, AlignJustify } from 'react-feather';
```
Comment (matmertz25) [inline src/components/SideBar.tsx:7]: It looks like `AlignJustify` might be an unused import

File: src/components/NavBar.tsx (hunk 1 above)
Comment (matmertz25) [reply src/components/NavBar.tsx:42]: I think if we are going to use a `div` tag here in stead of the AppBar component we need to remove the margin from the top of the `Dashboard` component content otherwise everything gets offset for me. 

https://github.com/ethersphere/bee-status/blob/34d2dfda5a89fc0b103d6aabbd2c1af5e06215a6/src/layout/Dashboard.tsx#L16

image

**Review by matmertz25 — APPROVED** [review]
Looks good, thanks for updating to the brand styling

**Comment by centerorbit** [comment]
Merging because these new styles/icons are merged into electron builds.
